db/config.cc

```diff
@@ -1398,7 +1408,7 @@ future<gms::inet_address> resolve(const config_file::named_value<sstring>& addre
         }
     }
 
-    co_return coroutine::exception(std::move(ex));
+    co_return seastar::coroutine::exception(std::move(ex));
```

Comment:
?

Reply:
The burden of including a header that invokes another header that has using namespace... we need to stop using that in headers, to avoid namespace collision.

Reply:
Ah, I'll patch it out.

Reply:
Ah, we can't, that's utils/coroutine.hh or something. Ok.